doc/riverctl.1.scd

```diff
@@ -76,6 +76,9 @@ used to control and configure river.
 	If the focused view is floating, make it tiled. If it is tiled, make
 	it floating.
 
+*importance* *up*|*down*
+	Move the focused view up or down in the view stack.
```

Comment:
Name suggestion: `swap`

Needs better documentation. While the sentence is not incorrect, it is better to describe what happens *on screen* rather than what happens in an internal data structure (which is totally opaque to the user). I propose "Swap the focused window with the next/previous visible non-floating window".

Reply:
`swap` works for me.

Reply:
> Although it would be nice if it would wrap.

Added wraping